README.md

```diff
@@ -91,30 +91,37 @@ Browser Mod has a number of services you can call to cause things to happen in t
 
 ## Popup card
 
-A popup card can be used to replace the more-info dialog of an entity with something of your choosing.
+A popup card can be used to replace the more-info dialog of an entity with something of your choosing. The entity can be targetted by entity, area, label or device.
 
-To use it, add a "Custom: Popup card" to a dashboard view via the GUI, pick the entity you want to override, configure the card and set up the popup like for the [`browser_mod.popup` service](documentation/services.md).
+To use it, add a "Custom: Popup card" to a dashboard view via the GUI, pick the target (entity, area, label, device) you want to override, configure the card and set up the popup like for the [`browser_mod.popup` service](documentation/services.md).
 
 The card will be visible only while you're in Edit mode.
 
 Custom popup cards are either local to the current Dashboard view (default) or can be used across all views of the Dashboard. Use the `Popup card is available for use in all views` GUI switch or `popup_card_all_views` optional parameter in Yaml. Using global view custom popup cards allows you to use a sub view to store your custom popup cards for a Dashboard, if that fits your use case.
 
+Using a wide target (label, area, device) and/or popup cards which are global to a view, allows for much customisation of more-info dialog to suit most cases.
+
 Yaml configuration:
 
 ```yaml
 type: custom:popup-card
-entity: <entity id>
+[entity: <entity_id> ]
+target:
+  [entity_id: <entity_id> ]
+  [area_id: <area-id>     ]
+  [label_id: <label-id>   ]
+  [device_id: <device-d>  ]
 card:
   type: ...etc...
 [popup_card_all_views: [true/FALSE]]
 [any parameter from the browser_mod.popup service call except "content"]
 ```
 
-> *Note:* It's advisable to use a `fire-dom-event` tap action instead as far as possible. Popup card is for the few cases where that's not possible. See [`services`](documentation/services.md) for more info.
```

Comment:
This comment confuses users. Now there is lots of flexibiliy with custom:popup-card I have removed this not.